aries-site/src/examples/templates/dashboards/DashboardExample.js

```diff
@@ -74,145 +74,172 @@ const data = [
-            {dashboardItem.message === 'Connected' && (
-              <Box round pad="xsmall" background="green" />
-            )}
-          </Footer>
-        </Box>
+      {dashboardItems.map(item => (
+        <Tile content={item} />
       ))}
     </Grid>
   );
 };
 
-const SidebarExample = ({ ...rest }) => {
+const Tile = ({ content }) => {
+  const cornerSize = 'small';
+
+  return (
+    <Box
+      round={cornerSize}
+      background={content.color}
+      key={`Tile ${content.title}`}
+      alignContent="center"
+      onClick={() => {}}
+    >
+      <Box
+        pad={{ horizontal: 'medium', vertical: 'small' }}
+        gap="medium"
+        flex
+        size="small"
+        direction="column"
+        align="start"
+      >
+        {content.icon}
+        <Box>
+          <Text size="medium" weight="bold">
+            {content.title}
+          </Text>
+          <Text size="medium">{content.subTitle}</Text>
+        </Box>
+      </Box>
+      <Footer
+        pad="small"
+        background="background-contrast"
+        round={{ corner: 'bottom', size: cornerSize }}
+      >
+        <Text size="xsmall">{content.message}</Text>
+        {content.message === 'Connected' && (
+          <Box round pad="xsmall" background="green" />
+        )}
+      </Footer>
+    </Box>
+  );
+};
+
+const Page = ({ children }) => {
+  return (
+    <Box flex overflow="auto">
+      <Box pad={{ horizontal: 'medium', bottom: 'large' }} flex="grow">
+        {children}
+      </Box>
+    </Box>
+  );
+};
+
+const AppSidebar = ({ ...rest }) => {
   const size = useContext(ResponsiveContext);
+
   return (
     <Sidebar
-      background="blue"
-      height={{ min: size !== 'small' ? '100%' : 'auto' }}
+      /* Sidebar should switch from column to row orientation 
+      when on smaller screens */
       direction={size !== 'small' ? 'column' : 'row'}
+      /* Only display most critical navigation items in mobile 
+      contexts */
+      header={
+        size !== 'small' && (
+          <Avatar
+            margin={{ bottom: 'medium' }}
+            src="//s.gravatar.com/avatar/b7fb138d53ba0f573212ccce38a7c43b?s=80"
+          />
+        )
+      }
+      footer={
+        size !== 'small' && (
+          <Box gap="medium">
+            <Button a11yTitle="Chat" icon={<Chat color="text-xweak" />} />
+            <Button
+              a11yTitle="Help"
+              icon={<StatusUnknown color="text-xweak" />}
+            />
+          </Box>
+        )
+      }
+      /* Min height is not needed in mobile contexts */
+      height={size !== 'small' ? { min: '100%' } : undefined}
+      align="center"
+      background="blue!"
       pad={{
-        horizontal: size !== 'small' ? 'small' : 'medium',
-        vertical: size !== 'small' ? 'medium' : 'small',
+        top: size !== 'small' ? 'medium' : 'small',
+        bottom: 'medium',
+        horizontal: size !== 'small' ? 'medium' : 'small',
```

Comment:
Feels like there's too much padding on this. Would expect it to be more like this: https://storybook.grommet.io/?path=/story/sidebar--tooltips

Reply:
Referencing padding of 24px based off of this: https://www.figma.com/file/xtpLd3KD9XfwfUPuKYKILW/HPE-Sidebar-Component?node-id=10%3A1